python/sglang/srt/layers/attention/flashinfer_backend.py

```diff
@@ -1004,6 +1055,26 @@ def call_begin_forward(
                 custom_mask=custom_mask,
                 non_blocking=True,
             )
+        elif (
+            global_config.enable_flashinfer_mla
+            and not global_server_args_dict["disable_radix_cache"]
```

Comment:
@zhyncs A quick question, I feel if I want to make MTP work with flashinfer backend. During the target verify stage, it is this code block has to be run? because perfill increment computation has to use the absorb trick with flashinfer?

Reply:
The full paged version is a temporary solution. I will soon support ragged prefill + paged prefill + paged decoding. These days have been quite busy, so I haven't updated.